cinema/serializers.py

```diff
@@ -68,13 +78,49 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    movie_session = MovieSessionListSerializer(read_only=True)
+
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session", "order")
```

Comment:
you don't need to display order on a ticket - in this task we actually display a ticket inside of the Order